tests/apollo/test_skvbc_chaotic_startup.py

```diff
@@ -48,6 +50,99 @@ class SkvbcChaoticStartupTest(unittest.TestCase):
 
     __test__ = False  # so that PyTest ignores this test scenario
 
+    @unittest.skip("Edge case scenario - not part of CI")
```

Comment:
can you also specify the BC-XXXX ticket number that this test allows to reproduce?

Reply:
AFAIK, the approach is to avoid mentioning ticket numbers since they are not public.